exonum/src/sandbox/consensus/transactions.rs

```diff
@@ -481,11 +481,11 @@ fn handle_tx_verify_signature() {
     let sandbox = timestamping_sandbox();
 
     // generate incorrect tx
-    let (public_key1, _) = gen_keypair();
-    let (_, secret_key2) = gen_keypair();
-
-    let data = vec![0; 64]; // TODO: Find the way how to get rid of hard-coded value. (ECR-1627)
-    let tx = TimestampTx::new(&public_key1, &data, &secret_key2);
+    let (public_key, _) = gen_keypair();
+    let (_, secret_key) = gen_keypair();
+    let tx = TimestampingTxGenerator::with_keypair(DATA_SIZE, (public_key, secret_key))
```

Comment:
Maybe it would be nice to indicate that pub and secret keys are not compatible like it was before. Like pub_key, wrong_sec_key, what do you think?